index.html

```diff
@@ -985,93 +985,49 @@ <h4>Deletion</h4>
-                                <li>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-id">
-                                        The collection object MUST contains an `id` attribute
-                                        pointing to the current page (self link).
-                                    </span>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-links">
-                                        The collection object MUST contains a `nextLink` attribute
-                                        for querying the next page, when present.
-                                    </span>
-                                    The links may be relative to directory API base URL, or be absolute URLs.
-                                    These links may include additional arguments for ordering or session keys.
-                                </li>
-                                <li>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-total">
-                                        When requested, the collection object MUST contain a `total` attribute 
-                                        equal to total number of TDs within the directory.
-                                    </span>
-                                </li>
-                                <li>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-order-default">
-                                        By default, the collection MUST be sorted alphanumerically 
-                                        by the unique identifiers of TDs.
-                                    </span>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-order">
-                                        The server MAY support sorting by other TD attributes (e.g. `created` field) and 
-                                        different sorting orders (i.e. ascendingly/descendingly).
-                                    </span>
-                                    <span class="rfc2119-assertion" id="tdd-reg-list-pagination-order-unsupported">
-                                        If the server does not support a given sorting argument, 
-                                        it MUST reject the request with 501 (Not Implemented) status.
-                                    </span>
-                                </li>
-                            </ul>
+                            Most HTTP/1.1 clients automatically process the data received with 
+                            chunked transfer encoding.
+                            Memory-constrained applications which require the full list
+                            should consider processing the received data incrementally.
+
+                            <p class="ednote" title="HTTP/2 chunking">
```

Comment:
OK, I see you do include this.  Good.  Yeah, testing is going to be a pain.  However, we probably DO need to include support for HTTP/2.

Reply:
My implementation supports both HTTP/1.1 and 2 (built-in Go HTTP library). I just need to prepare SSL certificates (generally required for HTTP/2) to and simply test it with a modern browser and curl.

Reply:
I finished testing this in the following ways:
-  HTTP/2 proxy in front of the directory. This was as simple as adding an http2 flag to the Nginx config. The proxy was able to translate HTTP/1.1 chunked encoding to an HTTP/2 stream. The HTTP/2 performance (speed) was twice better than HTTP/1.1 when tested over the internet.
- Adding TLS certificates on the directory. This enabled the serving HTTP/2 clients with the HTTP/2 stream. A local benchmarking showed similar performance to HTTP/1.1 over TLS. I guess the bottleneck on both HTTP request versions is the underlying database.

I will go ahead with taking the assertion out of the ednote.